spec/queries/all_vocabs_query_spec.rb

```diff
@@ -4,7 +4,7 @@
   describe ".call" do
     context "when vocabularies exist" do
       let(:vocabulary) { Vocabulary.new("bla") }
-      let(:sparql_client) { VocabularyInjector.new.sparql_client }
+      let(:sparql_client) { SPARQL::Client.new("#{Settings.marmotta.url}/sparql/select") }
```

Comment:
I tried using VocabularyInjector.new.sparql_client, but the spec fails to find the persisted vocabulary in the test context.